### PR TITLE
Facu - PODVI - Various enhancements

### PR DESCRIPTION
**Pull request by fmestevez**

### CODE REVIEW

File: public/js/play.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ var playState = {
         
         // Add player's animations
         this.player.animations.add('run', [0, 1], 8, true);
+        this.player.animations.add('walk', [0, 4, 3], 8, true);
```
Comment (Marthinoth) [inline public/js/play.js:17]: In the walk animation, should it be 5 instead of 3?

Comment (Marthinoth) [inline public/js/play.js:17]: Review complete. I have only one question, if no code change is needed, I'm ok merging this

Comment (fmestevez) [inline public/js/play.js:17]: That's what I first thought, but it seems the index is being assigned in some weird order

Comment (Marthinoth) [inline public/js/play.js:17]: Gotcha, the weird is that the jump animation is using also 3, but we haven't tested yet, it may be wrong indexed as well.

**Comment by Marthinoth** [comment]
Good. nice work Facu

**Pull request merged**